src/app.rs

```diff
@@ -29,15 +29,20 @@ use tui_input::Input;
 
 use crate::bpf_program::BpfProgram;
 
+const NUM_COLUMNS: i8 = 7;
```

Comment:
I see that we used to have a HEADER_COLUMNS constant in main.rs and then dropped in this PR. Can we bring that back and then ask its size? I worry that by having more than one place where we store the number of columns may lead to bugs.

Reply:
The main.rs `HEADER_COLUMNS` was moved into app.rs as `header_columns` field. This was so that the UI doesn't have to re-concat the header text and sort symbol in the draw loop. 
In the PR, app.rs now handles the text headers, which also handles applying the appropriate sort symbol to the selected header column. The main.rs now just renders the header text from the `header_columns` field in app.rs. 

But I updated it so that the `i8` const is no longer needed. It should now only rely on `header_columns.len()` when getting the size.